presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaThreadIdentifier.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.kafka;
+
+import com.facebook.presto.spi.HostAddress;
+
+import java.util.Objects;
+
+public class KafkaThreadIdentifier
+{
+    public final String threadId;
+    public final HostAddress hostAddress;
+    public final String partitionId;
```

Comment:
The partitionId is always a number, isn't it? why using a string type here?

Reply:
Good point. 

Also partitionId info was used as debug info in log.debug(). I don't think it's required here. Removed it for simplification.